### PR TITLE
fix: Print method to use `class_` #232

### LINKED ISSUE #232: Print method needs to use `class_` etc
Currently

```R
bizarro
#> <R7_generic> function (x, ...)  with 5 methods:
#> 1: method(bizarro, "integer")
#> 2: method(bizarro, "double")
#> 3: method(bizarro, "character")
#> 4: method(bizarro, new_S3_class("factor"))
#> 5: method(bizarro, "logical")
```

### PR DESCRIPTION
**Pull request by jamieRowen**
A fix to close #232 

``` R
bizarro
<R7_generic> bizarro(x, ...) with 5 methods:
1: method(bizarro, class_integer)
2: method(bizarro, class_double)
3: method(bizarro, class_character)
4: method(bizarro, new_S3_class("factor"))
5: method(bizarro, class_logical)
```
and with multiple-dispatch

``` R
<R7_generic> bizarro(x, y, ...) with 1 methods:
1: method(bizarro, list(class_character, new_S3_class("factor")))
```

Some of the test snapshots had to be updated for this. Since all cases of a base class for the primitive types are named `class_<X>` this simply appends `class_` to the beginning of the string on deparse.

### CODE REVIEW

File: R/class-spec.R [hunk 1]
```diff
@@ -161,7 +161,7 @@ class_deparse <- function(x) {
     any = "class_any",
     S4 = as.character(x@className),
     R7 = R7_class_name(x),
-    R7_base = encodeString(x$class, quote = '"'),
+    R7_base = paste0("class_", x$class), #encodeString(x$class, quote = '"'),
```
Comment (hadley) [inline R/class-spec.R:164]:
```suggestion
    R7_base = paste0("class_", x$class),
```

**Review by hadley — APPROVED** [review]
Looks good thanks! Can you please also add a short bullet to `NEWS.md`

File: R/class-spec.R (hunk 1 above)
Comment (jamieRowen) [reply R/class-spec.R:164]: Good spot, commented it out to test it worked and forgot to remove, thanks

**Comment by jamieRowen** [comment]
NEWS updated

**Pull request merged**